Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -530,9 +530,18 @@ baseType : BOOLEAN | BYTE | CURRENCY | DATE | DOUBLE | INTEGER | LONG | LONGLONG
 
 comparisonOperator : LT | LEQ | GT | GEQ | EQ | NEQ | IS | LIKE;
 
-complexType : expression;
+complexType :
```

Comment:
PC crashed. Looks fine then I guess? Haven't looked at the resolver part though.